scripts/run_pretty_printer_tests.py

```diff
@@ -0,0 +1,121 @@
+import os
+import platform
+import stat
+import sys
+from enum import Enum, auto
+from os import path
+
+import argparse
+
+from common import execute_command
+
+
+class OS(Enum):
+    Linux = auto()
+    MacOs = auto()
+    Windows = auto()
+    Other = auto()
+
+
+class Arhitecture(Enum):
+    X86_64 = "x64"
+    ARM64 = "aarch64"
+    Other = "other"
+
+
+# TODO: migrate to match
+def current_os() -> OS:
+    if sys.platform == "linux":
+        return OS.Linux
+    elif sys.platform == "darwin":
+        return OS.MacOs
+    elif sys.platform == "win32":
+        return OS.Windows
+    else:
+        return OS.Other
+
+
+def current_arhitecture() -> Arhitecture:
+    machine = platform.machine()
+    if machine == "x86_64" or machine == "AMD64":
+        return Arhitecture.X86_64
+    elif machine == "aarch64":
```

Comment:
```suggestion
    elif machine == "aarch64" or machine == "arm64":
```
To cover macOS (`arm64`) and Windows (`ARM64`, AFAIK)